test/end-to-end/Exception Handling/exception handling.ts

```diff
@@ -0,0 +1,11 @@
+-
+try {
+    // ...
+}
```

Comment:
I believe the standard for TypeScript would be to put `}` and continuations on the same line:

```typescript
try {
    // ...
} catch (error) {
    // ...
}